Polygon.js

```diff
@@ -0,0 +1,85 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?://(?:www\\.)polygon\\.com",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-12-05 20:54:17"
+}
+
+// http://en.wikipedia.org/wiki/User_talk:Czar
+
+function detectWeb(doc, url) {
+	if (doc.title.match("| Polygon")) {
+ 		return "webpage";
+ 	}
+	
+	return null;
+}
+
+function scrape(doc, url) {
+	var namespace = doc.documentElement.namespaceURI; 
+	var nsResolver = namespace ? function(prefix) {
+		if (prefix == 'x') return namespace; else return null; 
+		} : null; 
+	
+	var newItem = new Zotero.Item("webpage");
+	newItem.url = doc.location.href;
+	newItem.language = "en";
+	newItem.publicationTitle="Polygon";
+	
+		// Article title
+	var soughtXPath = "//h2";
+		evalXPath = doc.evaluate(soughtXPath, doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+	newItem.title = evalXPath;
+	
+		// Authors
+	var authorCounter = 0;
+	var authorArray = new Array();
+	do {
+		soughtXPath = "//div[@id='article-content']/header/p/a["+(authorCounter+1)+"]"
+		if (doc.evaluate(soughtXPath, doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext()) {
+			evalXPath = doc.evaluate(soughtXPath, doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+			//Zotero.debug(authorCounter+": "+evalXPath);
+			var firstName = evalXPath.substring(0,evalXPath.lastIndexOf(" "));
+			var lastName = evalXPath.substring(evalXPath.lastIndexOf(" ")+1);
+			authorArray.push({ firstName:firstName, lastName:lastName, creatorType:"author" });
+			authorCounter++;
+		}
+		else break;
+	}
+	while (evalXPath);
+	newItem.creators = authorArray;
+	
+		// Date
+	var soughtXPath = '//*[@id="article-content"]/header/p';
+		evalXPath = doc.evaluate(soughtXPath, doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+	newItem.date = evalXPath.split("on ")[1].split(/\n/)[0];
+	
+		// Article text
+	/** not sure why this isn't importing
+	var soughtXPath = '//*[@id="article-body"]/p[1]';
+		evalXPath = doc.evaluate(soughtXPath, doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+	newItem.abstract = evalXPath+" ..." ; **/
```

Comment:
this should be `newItem.abstracNote`
aurimasv.github.io/z2csl/typeMap.xml
has a full field list.

Again, consider if you can't scrape this from the metaheader.